.env

```diff
@@ -0,0 +1,4 @@
+APP_PORT=8000
+PROMETHEUS_PORT=9000
+NOTIFICATIONS_PORT=8085
+SOURCES_PORT=8131
```

Comment:
This file does not appear to be included anywhere, did you mean to commit it?

Reply:
added in docs
https://github.com/RHEnVision/provisioning-compose/pull/3/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R68

Reply:
Ah might be better to name it `ports.env` or something like that.

Problem is, you commit this into git therefore nobody else can change it without fighting against git. Would be perhaps better to create `ports.example.env` and give instruction to `cp ports.example.env ports.env`. The contents could be:

```
#APP_PORT=8000
#PROMETHEUS_PORT=9000
#NOTIFICATIONS_PORT=8085
#SOURCES_PORT=8131
```

All values commented so users could uncomment and work with the file as needed without having to fight git.